india_compliance/gst_india/report/hsn_wise_summary_of_outward_supplies/hsn_wise_summary_of_outward_supplies.py

```diff
@@ -221,8 +232,14 @@ def get_tax_accounts(
                         continue
                     itemised_tax.setdefault(item_code, frappe._dict())
                     if isinstance(tax_data, list):
+                        tax_rate = 0
+                        is_gst_tax = 0
+                        if account_head in output_gst_accounts:
+                            is_gst_tax = 1
+                            tax_rate = tax_data[0]
                         tax_amount = tax_data[1]
                     else:
+                        tax_rate = 0
                         tax_amount = 0
```

Comment:
It seems like `is_gst_tax` is not available in the else block. Define variables outside if blocks - change inside blocks.